core/app/c/[communitySlug]/forms/[formSlug]/edit/EditFormTitleButton.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import { useState } from "react";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+
+import { Button } from "ui/button";
+import { Dialog, DialogContent, DialogFooter, DialogTitle, DialogTrigger } from "ui/dialog";
+import {
+	Form,
+	FormControl,
+	FormDescription,
+	FormField,
+	FormItem,
+	FormLabel,
+	FormMessage,
+} from "ui/form";
+import { Input } from "ui/input";
+import { toast } from "ui/use-toast";
+
+import { didSucceed, useServerAction } from "~/lib/serverActions";
+import { updateForm } from "./actions";
+
+const editFormTitleSchema = z.object({
+	name: z.string().min(1, "Form name is required"),
+});
+
+const EditFormTitleButton = ({ name, communityId }: { name: string; communityId: string }) => {
+	const [isOpen, setIsOpen] = useState(false);
+
+	const form = useForm<z.infer<typeof editFormTitleSchema>>({
+		resolver: zodResolver(editFormTitleSchema),
+		defaultValues: {
+			name,
+		},
+	});
+
+	const runUpdateFormTitle = useServerAction(updateForm);
+	const onSubmit = (data: z.infer<typeof editFormTitleSchema>) => {
+		const result = runUpdateFormTitle({ communityId, name: data.name });
+		if (didSucceed(result)) {
+			toast({
+				title: "Success",
+				description: "Form name was successfully updated",
+			});
+			setIsOpen(false);
+		}
+	};
+	return (
+		<Dialog onOpenChange={setIsOpen} defaultOpen={false} open={isOpen} modal={true}>
+			<DialogTrigger asChild>
+				<Button
+					variant="link"
+					className="ml-2 text-sm text-gray-500 hover:text-blue-600 hover:underline"
```

Comment:
I think this is always blue in the designs, unless you know something I don't!

Reply:
the icon used was only on hover so i figured its blue on hover but i can ask katy

Reply:
https://www.figma.com/design/SqOTMElJn7fuPljfQL1gIY/PubPub-Platform?node-id=408-14041&m=dev
this is what i was going off of